doc/src/build/install.md

```diff
@@ -199,14 +198,14 @@ Sui requires the following additional tools on computers running Windows.
  * The [LLVM Compiler Infrastructure](https://releases.llvm.org/)
 
 >**Tip:** The installation progress might appear hanging if the `cmd.exe` window loses focus;
->press the `enter` key in the command prompt fix the issue.
+>press the `enter` key in the command prompt to fix the issue.
 
->**Known Issue:** The `sui console` command does not work in PowerShell.
+>**Known issue:** The `sui console` command does not work in PowerShell.
 
 
 ## Install Sui binaries
 
-After you install Cargo, use the following command to install Sui binaries:
+With Cargo installed, use the following command to install Sui binaries:
```

Comment:
The compiled binaries are now included with each release. We need to include info on how to use those as well